src/js/services/lang.js

```diff
@@ -67,7 +67,42 @@ const handleChangeLanguageNav = () => {
   }
 };
 
+// Request New Language form
+const languageRequest = () => {
+  const submitBtn = $('#submit-btn');
+  const langForm = $('.lang-form');
+  const input = $('#lang-req')[0];
```

Comment:
why is this at [0]?

Reply:
grabbing the element via ID using jQuery returns a jQuery object that contains the elements. If we want to get a specific DOM node with all the attributes that we are looking for, we have to specify which DOM node we want in order to access the attributes (in this case `.value`).

**This is the jQuery orbject**
<img width="385" alt="Screen Shot 2021-12-29 at 12 37 03 PM" src="https://user-images.githubusercontent.com/57234605/147693197-75db8edf-12e7-4bda-a4b3-1ea15f6cd65f.png">

**versus the actual DOM node we want** 
<img width="377" alt="Screen Shot 2021-12-29 at 12 37 12 PM" src="https://user-images.githubusercontent.com/57234605/147693231-73c0733e-0a1a-431f-968a-63f32fd5b81b.png">

Reply:
ah i see, id is usually a unique identifier so I was confused why it was coming as an array that you have to take the first element in the list because I was unsure why there would ever be more than 1

Reply:
why does #submit-btn not have [0] but #lang-req does?

Reply:
Maybe can use document.getElementById?

Reply:
In the case of `#submit-btn` we are just adding an event listener to it and then replacing it with a span when users submit a language. For `#lang-req` we have to access the value attribute of the DOM node within the jQuery object so we have specify the DOM node. 

I could use `document.getElementById` but my thought process was that we are using a lot of jQuery throughout and also jQuery methods so I wanted to keep it consistent.